### PR TITLE
Removing quotation marks from list-resources.sh

### LINKED ISSUE #77074: gci-gce-scalability - listResource fails
**Which jobs are failing**:
gci-gce-scalability and all other gce performance tests

**Which test(s) are failing**:
List resource fails.

**Since when has it been failing**:
Run 1121352546590396419. 02:57 PDT 04/25/2019

**Testgrid link**:
https://testgrid.k8s.io/sig-scalability-gce#gci-gce-scalability

**Reason for failure**:
```
To search the help text of gcloud commands, run:
gcloud help -- SEARCH_TERMS
Attempt 4 failed to list sinks. Retrying.
ERROR: (gcloud.logging.sinks.list) unrecognized arguments:  
```
After 4th attempt the script fails.

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by krzysied**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind bug

**What this PR does / why we need it**:
Fixes the cluster/gce/list-resources.sh script.

**Which issue(s) this PR fixes**:

Fixes #77074

**Special notes for your reviewer**:
Additional "" causes "" to be send as argument triggering unrecognized arguments error.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by krzysied** [comment]
ref https://github.com/kubernetes/kubernetes/pull/76881

File: cluster/gce/list-resources.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ function gcloud-list() {
   local attempt=1
   local result=""
   while true; do
-    if result=$(gcloud "${group}" "${resource}" list --project="${PROJECT}" "${filter:+--filter="$filter"}" "${@:4}"); then
+    if result=$(gcloud "${group}" "${resource}" list --project="${PROJECT}" ${filter:+--filter="$filter"} ${@:4}); then
```
Comment (fejta) [inline cluster/gce/list-resources.sh]: I would expect that we still want `"${@:4}"`

Comment (krzysied) [reply cluster/gce/list-resources.sh]: Done.

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**